unit-tests/live/options/test-rgb-options-metadata-consistency.py

```diff
@@ -90,7 +94,10 @@ def check_option_and_metadata_values(option, metadata, value_to_set, frame):
             iteration = (iteration + 1) % (3 * (num_of_frames_to_wait + 1))
 
         except:
+            close_resources(color_sensor)
             test.unexpected_exception()
```

Comment:
This won't abort -- why do we want to close_resources before it?

Reply:
I thought it aborts, I will change it.